tests/foreman/cli/test_subnet.py

```diff
@@ -115,7 +115,7 @@ def test_positive_CRUD(module_target_sat):
     pool = sorted(valid_addr_pools()[0])
     # generate pool range from network address
     new_network = gen_ipaddr()
-    new_mask = gen_netmask()
+    new_mask = '255.255.255.0'
```

Comment:
Can we keep old netmask values before mentioned PR as is, and new_mask should be different than mask value here?
```suggestion
    new_mask = '255.255.192.0'
```

Reply:
Sure.